tests/testthat/_snaps/shuffle_ard.md

```diff
@@ -432,3 +432,133 @@
       1 Overall ARM AGE      <NA>       p          0.05
       2 <NA>        AGE      continuous mean      75.1 
 
+---
```

Comment:
Since this snapshot splits into two chunks, I think we can add something like this above the snapshot call to increase the width. `withr::local_options(list(width = 90))`

Reply:
that worked beautifully!